src/sentry/snuba/errors.py

```diff
@@ -177,9 +181,9 @@ def timeseries_query(
         col_name = base_builder.aggregates[0].alias
         # If we have two sets of results then we're doing a comparison queries. Divide the primary
         # results by the comparison results.
-        for result, cmp_result in zip(results[0]["data"], results[1]["data"]):
+        for ret_result, cmp_result in zip(results[0]["data"], results[1]["data"]):
             cmp_result_val = cmp_result.get(col_name, 0)
-            result["comparisonCount"] = cmp_result_val
+            ret_result["comparisonCount"] = cmp_result_val
```

Comment:
Was this change necessary?

Reply:
`result` clobbers several (immutable) locals